.github/CODEOWNERS

```
@@ -803,6 +803,7 @@ lib/bid @department-of-veterans-affairs/disability-benefits @department-of-veter
 lib/bpds @department-of-veterans-affairs/benefits-lifestage @department-of-veterans-affairs/bpds @department-of-veterans-affairs/backend-review-group
 lib/carma @department-of-veterans-affairs/vfs-10-10 @department-of-veterans-affairs/backend-review-group
 lib/caseflow @department-of-veterans-affairs/lighthouse-banana-peels @department-of-veterans-affairs/backend-review-group
+lib/ccra/configuration.rb @department-of-veterans-affairs/backend-review-group
```

Your team will still want to own this, right?
```suggestion
lib/ccra/configuration.rb @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/vfs-vaos
```

---

Yes on both, thanks!